python/cli/prelude_cli/views/iam.py

```diff
@@ -102,117 +117,128 @@ def detach_oidc(controller):
 @click.pass_obj
 @pretty_print
-def describe_account(controller):
-    """Get account details"""
-    with Spinner(description="Fetching account details"):
-        return controller.get_account()
+def get_oidc_name(controller, slug):
+    """Get OIDC provider name from organization slug"""
+    with Spinner(description="Fetching OIDC provider name"):
+        return controller.get_oidc_name(slug=slug)
 
 
-@iam.command("create-user")
-@click.option(
-    "-d", "--days", help="days this user will remain active", default=365, type=int
-)
+@iam.command("invite-user")
 @click.option(
     "-p",
     "--permission",
     help="user permission level",
-    default=Permission.SERVICE.name,
+    default=Permission.EXECUTIVE.name,
     type=click.Choice(
-        [p.name for p in Permission if p != Permission.INVALID], case_sensitive=False
+        [
+            p.name
+            for p in Permission
+            if p not in [Permission.INVALID, Permission.SERVICE, Permission.SUPPORT]
+        ],
+        case_sensitive=False,
     ),
     show_default=True,
 )
 @click.option(
     "-n", "--name", help="name of user", default=None, show_default=False, type=str
 )
-@click.option("-o", "--oidc", help="whether user must login via SSO", is_flag=True)
+@click.option(
+    "-o", "--oidc", help="OIDC app name", default=None, show_default=False, type=str
+)
 @click.argument("email")
 @click.pass_obj
 @pretty_print
-def create_user(controller, days, permission, name, oidc, email):
-    """Create a new user in your account"""
-    with Spinner(description="Creating new user"):
-        data = controller.create_user(
+def invite_user(controller, permission, name, oidc, email):
+    """Invite a new user to your account"""
+    with Spinner(description="Inviting new user"):
+        data = controller.invite_user(
             email=email,
+            oidc=oidc,
             permission=Permission[permission],
             name=name,
-            expires=datetime.now(timezone.utc) + timedelta(days=days),
-            oidc=oidc,
         )
     msg = "New user must check their email to verify their account"
-    return data, msg if permission != Permission.SERVICE.name else None
+    return data, msg
 
 
-@iam.command("reset-password")
+@iam.command("create-service-user")
 @click.argument("email")
-@click.option(
-    "-a",
-    "--account",
-    help="override the profile account id",
-    default=None,
-    show_default=False,
-    type=str,
-)
 @click.pass_obj
 @pretty_print
-def reset_password(controller, email, account):
-    """Reset a user's password"""
-    with Spinner(description="Resetting password"):
-        controller.reset_password(email=email, account_id=account)
-
-    token = click.prompt(
-        "Check your email for the reset token.\nEnter your reset token", type=str
-    )
-    with Spinner(description="Retrieving new password"):
-        return controller.verify_user(token=token)
+def create_service_user(controller, email):
```

Comment:
let's call it "handle" instead of "email" (since service users are often not emails)
also rm the "check your email to verify" msg